.gitignore

```diff
@@ -1,11 +1,14 @@
 *~
 *.pyc
 *.DS_Store
-/docs/*/build/*
+/docs/*build/*
```

Comment:
Why this? I have my docs build now unignored

Reply:
Isn't it suppose to be ignored? I believe repository should have visible build/v1/source and build/v2/source.